ruby-challenges.rb

```diff
@@ -10,6 +10,20 @@
 reposts3 = 221
 # Expected output: '221 is odd'
 
+def odd_or_even(num)
+  if num % 2 == 0
+    puts "#{num} is even"
+  else
+    puts "#{num} is odd"
+  end
+end
+
+
+p odd_or_even(reposts1)
+p odd_or_even(reposts2)
+p odd_or_even(reposts3)
+
+# I keep getting nil under my answer. I hypothosize that its due to the string interpolation depending on what value you apply to the method.
```

Comment:
This is because you have puts in your method as well as p when you perform your method calls on lines 22-24. Make sure to get rid of any "puts" or "p" within your methods.

Reply:
Will do.